components/admin_console/admin_definition.jsx

```diff
@@ -513,13 +513,13 @@ export default {
                             label: 'admin.log.enableWebhookDebugging',
                             label_default: 'Enable Webhook Debugging:',
                             help_text: 'admin.log.enableWebhookDebuggingDescription',
-                            help_text_default: 'To output the request body of incoming webhooks to the console, enable this setting and set {boldedConsoleLogLevel} to "DEBUG". Disable this setting to remove webhook request body information from console logs when in DEBUG mode.',
+                            help_text_default: 'To output the request body of incoming webhooks to the log file, enable this setting and set {boldedFileLogLevel} to "DEBUG". Disable this setting to remove webhook request body information from the log file when in DEBUG mode.',
```

Comment:
It's not necessarily to a log file. It's just to the logging system. Where that goes is determined by other settings. Instead of "the log file" just "the logs" would work. Then just specify the messages are sent at "DEBUG level" and that you might need to change the output level to see the messages.

Reply:
How about this

```
When true, sends webhook debug messages to the server logs. To also output the request body of incoming webhooks, set {boldedFileLogLevel} to "DEBUG".
```

Describes what happens when the settings is enabled, then says you have to change the log level to "DEBUG" to see the webhook request body.

Reply:
boldedFileLogLevel -> boldedLogLevel
Then it should be good.